extensions/amp-analytics/0.1/sandbox-vars-whitelist.js

```diff
@@ -49,4 +49,5 @@ export const SANDBOX_AVAILABLE_VARS = {
   'FIRST_CONTENTFUL_PAINT': true,
   'FIRST_VIEWPORT_READY': true,
   'MAKE_BODY_VISIBLE': true,
+  'DOCUMENT_REFERRER': true,
```

Comment:
Copying my comment from above:

> amp-skimlink needs to send the referrer within the tracking request but I am not really sure if this is ok do add DOCUMENT_REFERRER to the SANDBOX_AVAILABLE_VARS whitelist. If not I can still get the referrer from Javascript instead of using the Macro.
```

Reply:
Ah sorry missed that comment. Yes, can you try using JS instead?

Reply:
The idea is to limit the scope of `<amp-analytics>` from component versus of from publishers. In general document referral sounds ok and reasonable, but I believe we should still have a quick privacy review on that. 
In the meanwhile, can you try using JS? Thanks.